### PR TITLE
errors(ticdc): fix a typo and remove useless function

### LINKED ISSUE #5765: A typo in errors.go
### What did you do?

found a typos in errors.go file.

### What did you expect to see?

no typo

### What did you see instead?

typo

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
(paste TiCDC version here)
```

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #5765 

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by hi-rustin — APPROVED** [review]

**Review by CharlesCheung96 — APPROVED** [review]

File: errors.toml [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ unmarshal failed
 
 ["CDC:ErrUpStreamNotFound"]
 error = '''
-upStram not found, cluster-id: %d
+upStream not found, cluster-id: %d
```
Comment (overvenus) [inline errors.toml:1066]:
```suggestion
upstream not found, cluster-id: %d
```

"upstream" is a valid word, please do not camelCase.

Comment (asddongmen) [reply errors.toml]: done

**Review by overvenus — APPROVED** [review]

**Pull request merged**